packages/build-ui/src/components/button.rs

```diff
@@ -49,3 +55,49 @@ pub fn PrimaryButton(
         }
     }
 }
+
+
+
+#[component]
+pub fn ButtonWithIcon(
+    #[props(default = "".to_string())] class: String,
+    label: String,                                   
+    icon: Element,
+    #[props(default = true)] disabled: bool,                                   
+    onclick: EventHandler<MouseEvent>,              
+) -> Element {
+
+    rsx! {
+        button {
+            class: format!(
+                "border border-white text-white px-6 py-2 flex items-center justify-center bg-black active:bg-primary/25 active:border-primary hover:bg-primary/25 hover:border-primary disabled:border-neutral-80 disabled:text-neutral-80 disabled:bg-btn-disable{}",
```

Comment:
```suggestion
                "border border-white text-white px-6 py-2 flex items-center justify-center bg-black active:bg-primary/25 active:border-primary hover:bg-primary/25 hover:border-primary disabled:border-neutral-80 disabled:text-neutral-80 disabled:bg-btn-disable {}",
```

you may intend this, right?